Doppler.ContactPolicies.Data.Access/Repositories/ContactPoliciesSettings/ContactPoliciesSettingsRepository.cs

```diff
@@ -46,6 +47,53 @@ public async Task<Entities.ContactPoliciesSettings> GetContactPoliciesSettingsAs
             return contactPoliciesSettings;
         }
 
+        public async Task<Entities.ContactPoliciesSettings> GetBasicContactPoliciesSettingsAsync(string accountName)
+        {
+            using var connection = await _databaseConnectionFactory.GetConnection();
+            const string query =
+                @"select cp.IdUser, cp.Active, cp.Interval [IntervalInDays], cp.Amount [EmailsAmountByInterval], u.Email [AccountName]
+                from [User] u
+                left join [UserShippingLimit] cp on u.IdUser = cp.IdUser and cp.Enabled = 1
+                where u.Email = @Email;";
```

Comment:
Just a detail to continue with naming conventions
```suggestion
                @"select usl.IdUser, usl.Active, usl.Interval [IntervalInDays], usl.Amount [EmailsAmountByInterval], u.Email [AccountName]
                from [User] u
                left join [UserShippingLimit] usl on u.IdUser = usl.IdUser and usl.Enabled = 1
                where u.Email = @Email;";
```